tests/views/test_plot_view.py

```diff
@@ -112,3 +118,42 @@ def test_clearing_ensembles_view(
     assert len(chosen_parameters) == 0
 
     assert dash_duo.get_logs() == [], "browser console should contain no error"
+
+
+def test_axis_labels(mock_data, dash_duo):
+    """test_axis_labels loads two different plots in the plot view and checks
+    that axes are labelled correctly"""
+
+    plugin = setup_plugin(
+        dash_duo, __name__, ResponseComparison, window_size=(1024, 2048)
+    )
+
+    # find the right ensemble which has mock data prepared for this test
+    wanted_ensemble_name = "default3"
```

Comment:
I guess use the constant defined in `tests/data/snake_oil_data.py`

Reply:
thanks for catching that, fixed!